sdk/storage/Microsoft.Azure.Management.Storage/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Storage management functions for managing the Microsoft Azure Storage service.")]
 
 [assembly: AssemblyVersion("22.0.0.0")]
-[assembly: AssemblyFileVersion("22.0.0.0")]
+[assembly: AssemblyFileVersion("22.6.0.0")]
```

Comment:
Same. Need to be consistent in version number.

Reply:
Fixed. Thanks for the catch!